### DISCUSSION TITLE
v1.16

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-AutoUpdate#262] [Announcements]
Hi all,

Thanks to be more and more to use WAU.

I'd like to thank all of contributors and mainly @KnifMelti for the hard involve in this project and the big improvements and new features in the release he made.
Still in beta for now due to big big changes. Feel free to reports bugs before GA.

**Comment by KnifMelti** [comment]
Is it only me, or...
Shouldn't we see `$WAUVersion = "1.16.1" `in **Winget-AutoUpdate-Install.ps1** from https://github.com/Romanitho/Winget-AutoUpdate/releases/tag/v1.16.1 ?
It's still **1.15.3**...

**Reply by Romanitho** [reply]
You're right !

**Reply by Romanitho** [reply]
I always forgot to maintain the script itself version 🤯
